### PR TITLE
oVirt part 7

### PR DESCRIPTION
**Pull request by jortel**
oVirt part 7.

Renamed const to match [Event Documentation](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.2/html/technical_reference/appe-event_codes).  A lot more event to follow in future PR.

Improved handling of the oVirt data reconciler lifecycle.

Updated oVirt trees.
Renamed the /tree/host to /tree/cluster since the tree really needs to be focused on the cluster.  Putting the VM under the host reflects an untrue containment relationship.

Tree:
- DataCenter
  - Cluster
  - VM

The `workload` has been flattened to match.

Workload:
- ID
- Name
- ...
- Host
- Cluster
- DataCenter

### CODE REVIEW

File: pkg/controller/provider/container/ovirt/model.go [hunk 1]
```diff
@@ -11,22 +11,16 @@ import (
...
-	VmDeleted         = 113
+	USER_ADD_CLUSTER    = 809
+	USER_UPDATE_CLUSTER = 811
+	USER_REMOVE_CLUSTER = 813
+	USER_ADD_HOST       = 42
+	USER_UPDATE_HOST    = 43
+	USER_REMOVE_HOST    = 44
+	USER_ADD_VM         = 34
+	USER_UPDATE_VM      = 35
+	USER_REMOVE_VM      = 113
```
Comment (jortel) [inline pkg/controller/provider/container/ovirt/model.go:23]: This change is unrelated to flattened trees.
Using all CAPS to match [documentation](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.2/html/technical_reference/appe-event_codes)

**Review by mansam — APPROVED** [review]
LGTM
